resources/public/api/conf/1.0/application.yaml

```diff
@@ -131,8 +131,8 @@ paths:
             application/json:
               example:
                 {
-                  "code": "NOT_ACCEPTABLE",
-                  "message": "Cannot produce an acceptable response",
+                  "code": "ACCEPT_HEADER_INVALID",
```

Comment:
There are two possible errors here. Accept head not being valid, but also Content Type header not being valid.

Reply:
Would this be a more accurate message: 
Cannot produce an acceptable response. The Accept or Content-Type header is missing or invalid.

Reply:
Yes that would be good. The code should still be NOT_ACCEPTABLE though